migrations/versions/2021-01-04T20-23-50Z_dece77ed5036_init_codewars_challenge.py

```diff
@@ -0,0 +1,57 @@
+"""init codewars challenge
+
+Revision ID: dece77ed5036
+Revises: 2ba5cc3efce8
+Create Date: 2021-01-04 20:23:50.479134+00:00
+
+"""
+import datetime
+
+from alembic import op
+import sqlalchemy as sa
+
+from pyslackersweb.sirbot.models import codewars
```

Comment:
I mentioned this in slack, but figure we should place it here for posterity:

We can't use imported models in our migrations like this. The reason being is if the model ever is moved/renamed/deleted or columns are changed we won't be able to apply this migration to a new database. (This is actually a really nice thing with Django where you can use the model version in the migration, though it cannot be imported directly).

A few ideas:

1. We can add a temporary task or CLI target to run the data seed that we will remove later
2. Design a "seed" system that is always up-to-date with the latest version of the code
3. Manually seed these
4. Add a command to slack to add these
5. Duplicate the model definition to this file to make sure it's static as of this migration.

My ask is to remove this import and we will need to find another way to seed the challenges.

Reply:
Fixed by including the model in the migration.

I think it's good enough for now. If this feature get some traction we can add some more involved path to include new challenges.